public/app/plugins/datasource/cloudwatch/metric-math/language.ts

```diff
@@ -92,6 +93,7 @@ export const language: monacoType.languages.IMonarchLanguage = {
     variables: [
       [/\$[a-zA-Z0-9-_]+/, 'variable'], // $ followed by any letter/number we assume could be grafana template variable
     ],
+    macros: [[/\$__[a-zA-Z0-9-_]+/, 'type']],
```

Comment:
maybe  a comment, my brain was like "oh this is exactly the same as above and then I realized no there are underscores lol